### ISSUE TITLE
lua error and crash at runtime

### ISSUE THREAD

**Issue by cyberic99** [sasq64/chipmachine#65]
Hello!

I am trying to build chipmachine inside a docker container running debian sid
it crashes when I launch it:
```
gdb --arg ./chipmachine -X -d
```
```
[MusicDatabase.cpp:474] Workdir:"/home/builder/chipmachine"
[New Thread 0x7ffff21ce700 (LWP 187)]
[MusicDatabase.cpp:441] Checking Radio
[Thread 0x7ffff21ce700 (LWP 187) exited]
Creating 'Radio' database
[MusicDatabase.cpp:474] Workdir:"/home/builder/chipmachine"
[New Thread 0x7ffff21ce700 (LWP 188)]
[MusicDatabase.cpp:441] Checking Bitar till Kaffet
[Thread 0x7ffff21ce700 (LWP 188) exited]
Creating 'Bitar till Kaffet' database
[MusicDatabase.cpp:474] Workdir:"/home/builder/chipmachine"
[New Thread 0x7ffff21ce700 (LWP 189)]
[MusicDatabase.cpp:441] Checking Bitar till Kaffet
[Thread 0x7ffff21ce700 (LWP 189) exited]
Creating 'Bitar till Kaffet' database
[MusicDatabase.cpp:474] Workdir:"/home/builder/chipmachine"
[New Thread 0x7ffff21ce700 (LWP 190)]
[MusicDatabase.cpp:441] Checking This Week in Chiptune
[Thread 0x7ffff21ce700 (LWP 190) exited]
Creating 'This Week in Chiptune' database
[MusicDatabase.cpp:474] Workdir:"/home/builder/chipmachine"
[New Thread 0x7ffff21ce700 (LWP 191)]
[web.h:258] target: /home/builder/.cache/chipmachine/_webfiles/http%3a%2f%2fthisweekinchiptune.libsyn.com/rss
[MusicDatabase.cpp:441] Checking Gamewave Podcast
[Thread 0x7ffff21ce700 (LWP 191) exited]
Creating 'Gamewave Podcast' database
[MusicDatabase.cpp:474] Workdir:"/home/builder/chipmachine"
[New Thread 0x7ffff21ce700 (LWP 192)]
[web.h:258] target: /home/builder/.cache/chipmachine/_webfiles/http%3a%2f%2fgamewave.yays.co/rss.xml
[sol2] An exception occurred: 
[Thread 0x7ffff21ce700 (LWP 192) exited]
terminate called after throwing an instance of 'sol::error'
  what():  lua: error: 

Thread 1 "chipmachine" received signal SIGABRT, Aborted.
__GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
51	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
(gdb) up
#1  0x00007ffff65b22f1 in __GI_abort () at abort.c:79
79	abort.c: No such file or directory.
(gdb) 
#2  0x00007ffff6972943 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
(gdb) 
#3  0x00007ffff6978896 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
(gdb) 
#4  0x00007ffff69788d1 in std::terminate() () from /lib/x86_64-linux-gnu/libstdc++.so.6
(gdb) bt
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff65b22f1 in __GI_abort () at abort.c:79
#2  0x00007ffff6972943 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff6978896 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff69788d1 in std::terminate() () from /lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff6978b04 in __cxa_throw () from /lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00005555557edc01 in sol::default_at_panic (L=<optimized out>)
    at /usr/include/c++/8/ext/new_allocator.h:86
#7  0x00007ffff7d7f40a in ?? () from /lib/x86_64-linux-gnu/liblua5.2.so.0
#8  0x00007ffff7d7ef99 in ?? () from /lib/x86_64-linux-gnu/liblua5.2.so.0
#9  0x00007ffff7d7c3c9 in lua_error () from /lib/x86_64-linux-gnu/liblua5.2.so.0
#10 0x00005555557e205c in sol::stack::script (mode=sol::load_mode::any, chunkname="", code=..., 
    L=0x55555b3fd650) at /home/builder/chipmachine/src/../sol2/sol.hpp:11384
#11 sol::state_view::unsafe_script (mode=sol::load_mode::any, this=0x7fffffffdd48, chunkname="", code=...)
    at /home/builder/chipmachine/src/../sol2/sol.hpp:20779
#12 sol::state_view::script (mode=sol::load_mode::any, chunkname="", code=..., this=0x7fffffffdd48)
    at /home/builder/chipmachine/src/../sol2/sol.hpp:20846
#13 chipmachine::MusicDatabase::initFromLua(std::experimental::filesystem::v1::__cxx11::path const&) ()
    at /home/builder/chipmachine/src/MusicDatabase.cpp:1238
#14 0x000055555585f08f in chipmachine::ChipInterface::ChipInterface (this=0x55555667f020 <main::ci>, 
    wd=filesystem::path "/home/builder/chipmachine" = {...})
    at /home/builder/chipmachine/src/MusicDatabase.h:203
#15 0x00005555557c8782 in main () at /usr/include/c++/8/bits/shared_ptr_base.h:1145
#16 0x00007ffff659db17 in __libc_start_main (main=0x5555557c77a0 <main>, argc=3, argv=0x7fffffffe6e8, 
    init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7fffffffe6d8)
    at ../csu/libc-start.c:310
#17 0x00005555557d107a in _start () at /usr/include/c++/8/ext/new_allocator.h:79
(gdb) 

```
```
$ cat /etc/issue
Debian GNU/Linux buster/sid \n \l
```
```
dpkg -l|grep lua
ii  liblua5.2-0:amd64           5.2.4-1.1+b2                 amd64        Shared library for the Lua interpreter version 5.2
ii  liblua5.2-dev:amd64         5.2.4-1.1+b2                 amd64        Development files for the Lua language version 5.2
```
git commit: 3dc81cde72c3a8e23c716ad730ade6f966b5264b

Thx

**Comment by kg6hum** [comment]
I am having the same issue, but I am not building a docker container. This is on Ubuntu 18.04.01 with Lua 5.3.3.

**Comment by sasq64** [comment]
Should be fixed by 0e507e7596a
